### PR TITLE
Add ability to get it working with proxy

### PR DESCRIPTION
**Pull request by sethumadhav07**
currently slack connection using webhook is not working with proxy which is a common requirement in many corporate settings.
Code change in this PR addresses that.

### CODE REVIEW

**Comment by sethumadhav07** [comment]
@8398a7 can you please take a look at this PR?

**Comment by 8398a7** [comment]
@sethumadhav07 Sorry for the late confirmation.
CI doesn't seem to work for some reason. Will you push it again?

**Comment by sethumadhav07** [comment]
@8398a7 can you please take a look at it and approve?

File: dist/index.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
...rts=createHttpsProxyAgent},595:function(e,t,r){"use strict";var s=this&&this.__importDefault||function(e){return e&&e.__esModule?e:{default:e}};Object.defineProperty(t,"__esModule",{value:true});const o=s(r(8237));const n=o.default("https-proxy-agent:parse-proxy-response");function parseProxyResponse(e){return new Promise(((t,r)=>{let s=0;const o=[];function read(){const t=e.read();if(t)ondata(t);else e.once("readable",read)}function cleanup(){e.removeListener("end",onend);e.removeListener("error",onerror);e.removeListener("close",onclose);e.removeListener("readable",read)}function onclose(e){n("onclose had error %o",e)}function onend(){n("onend")}function onerror(e){cleanup();n("onerror %o",e);r(e)}function ondata(e){o.push(e);s+=e.length;const r=Buffer.concat(o,s);const i=r.indexOf("\r\n\r\n");if(i===-1){n("have not received end of HTTP headers yet...");read();return}const a=r.toString("ascii",0,r.indexOf("\r\n"));const c=+a.split(" ")[1];n("got proxy server response: %o",a);t({statusCode:c,buffered:r})}e.on("error",onerror);e.on("close",onclose);e.on("end",onend);read()}))}t.default=parseProxyResponse},3287:(e,t)=>{"use strict";Object.defineProperty(t,"__esModule",{value:true});
```
Comment (8398a7) [inline dist/index.js:1]: Do not commit this file.

**Review by 8398a7 — CHANGES_REQUESTED** [review]

File: dist/index.js (hunk 1 above)
Comment (sethumadhav07) [reply dist/index.js]: addressed please take a look

**Comment by 8398a7** [comment]
CI is failing due to reduced test coverage. Can you please fix it?

**Comment by sethumadhav07** [comment]
I fixed the code coverage for tests. can you please a look at it and approve?

**Review by 8398a7 — APPROVED** [review]

**Comment by sethumadhav07** [comment]
will a new release/tag be created?

**Comment by sethumadhav07** [comment]
@8398a7  can a new release/tag be created?

**Pull request merged**